### DISCUSSION TITLE
Bug? - Standard price rules aren't showing on backend but are showing on frontend.

### DISCUSSION THREAD

**Discussion by JoeMatkin** [swellstores/community#497] [Help] [answered]
The standard pricing rules don't show on the backend, when you fill in the data it then shows on the frontend but if you refresh the Swell backend the data doesn't show.

<img width="1172" alt="Screenshot 2024-02-07 at 09 09 28" src="https://github.com/swellstores/community/assets/72705313/0165a6dd-85a9-4730-84a5-3cbb1d48dfa3">

<img width="721" alt="Screenshot 2024-02-07 at 09 10 04" src="https://github.com/swellstores/community/assets/72705313/c2736fa0-d626-4edd-891c-a936aed24cec">

**Comment by logeshswell** [comment]
What do you mean backend and frontend here?

Do you mean that Standard pricing rules are not shown properly in the Swell-admin dashboard, but the values are being returned correctly in the [frontend APIs](https://developers.swell.is/frontend-api/products)?

If so, I couldn't find this issue in my store. Can you share your product URL from the Swell-admin dashboard?

Please correct me if I misunderstood, so that I can assist you accordingly.

**Reply by JoeMatkin** [reply]
https://x-distributions.swell.store/

It's for all products, backend meaning the Swell dashboard. Frontend being my website.

**Reply by logeshswell** [reply]
Does your website use Swell frontend or backend API?

> It's for all products

Can you share a sample product URL for us to investigate?

**Reply by logeshswell** [reply]
I could reproduce this issue in your store for [this sample product](https://x-distributions.swell.store/admin/products/65a7d27673399f0013815128).
Let me check with my team and keep you posted if I get some updates.

**Reply by logeshswell** [reply, marked as answer]
It is identified that this issue is happening only in GBP currency but not in other currency.

I can confirm that it's a bug at our end and I will keep you posted if I get any updates on the fix.

**Reply by JoeMatkin** [reply]
Hi,

Is there any update on this? It's a major issue for myself and my client.

Thanks

**Reply by logeshswell** [reply]
No updates at the moment. I will keep you posted if any updates are available.

**Reply by logeshswell** [reply]
This bug is fixed and deployed at our end. Can you check and confirm please?

**Reply by JoeMatkin** [reply]
I can confirm it's fixed, thanks!
